packages/search/addon/services/navi-base-search-provider.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Base search provider service.
+ */
+
+import Service from '@ember/service';
+import { assert } from '@ember/debug';
+
+export default class NaviBaseSearchProviderService extends Service {
+  /**
+   * @property {String} displayComponentName
+   */
+  displayComponentName;
```

Comment:
Can we make this a `get displayComponent..`? And assert that its defined in the subclass like search?